src/lib/containers/oauth/OAuthManagement.tsx

```diff
@@ -87,6 +99,38 @@ export const OAuthManagement: React.FunctionComponent = () => {
           isShowingConfirmModal={isShowingConfirmModal}
         />
       )}
+      <Modal
+        show={isShowingVerification}
+        animation={false}
+        backdrop="static"
+        onHide={() => setIsShowingVerification(false)}
+      >
+        <Modal.Header closeButton>
+          <Modal.Title>
+            <Typography variant="headline1">Submit Verification</Typography>
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <p>
+            In order to verify an OAuth client please submit a request to the{' '}
+            <a href="https://sagebionetworks.jira.com/servicedesk/customer/portal/9">
```

Comment:
Would it make sense to open this in a new window?  If you agree, please add
target="_blank"
rel="noopener noreferrer"